### ISSUE TITLE
The wiki is missing images

### ISSUE THREAD

**Issue by jonascj** [foreshadow/atom-python-run#118]
Thank you for an awesome package!

The wiki appears to be missing images, e.g. https://github.com/foreshadow/atom-python-run/wiki/12-How-Do-I-Use-atom-python-run

It is quite hard to follow some of the descriptions without the screenshots.

**Comment by foreshadow** [comment]
Hi, thank you for the reminder. There was someone who primarily maintained the wiki page. But he seems to have deleted his account in github and I cannot find him anyway. I don't have the origin images in the wiki and this is difficult to be fixed in a short time. I will try to fix them when I have time, maybe months later(as you can see I haven't updated this package for a long time too).

P.S. for everyone. I really want to hear from this contributer again. Maybe he was called @xovertheyearsx. I wonder whether he deleted his account because he don't want his contribution appears in this repo any longer (a reasonable guess, because these images uploaded by him are no longer available) or because of some other unrelated reasons.
